examples/drivers/windows/otLwf/iocontrol.c

```diff
@@ -1725,6 +1726,110 @@ otLwfTunIoCtl_otMasterKey_Handler(
 
 _IRQL_requires_max_(PASSIVE_LEVEL)
 NTSTATUS
+otLwfIoCtl_otPSKc(
+    _In_ PMS_FILTER         pFilter,
```

Comment:
As per style guide, function arguments should start with a lower case `a` to denote the scope